config/yurtctl-servant/setup_edgenode

```diff
@@ -119,11 +120,18 @@ setup_yurthub() {
         log "setting up yurthub on nodes of minikube"
         yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
             sed 's|__pki_path__|/var/lib/minikube/certs|')
```

Comment:
yurthub does not need kubeconfig file of kubelet, because yurthub generates certificate itself based on join token. so i think we delete the following setting.
1. __pki_path__ in yurthub template
2. volumes setting except add a new /var/lib/yurthub volume

Reply:
Adjusted